iamclient/iamclient_test.go

```diff
@@ -517,31 +570,31 @@ func (server *testServer) UnregisterService(
 	return rsp, nil
 }
 
-func (server *testServer) GetPermissions(
+func (server *testPublicServer) GetPermissions(
 	ctx context.Context, req *pb.PermissionsRequest) (rsp *pb.PermissionsResponse, err error) {
 	rsp = &pb.PermissionsResponse{}
 
-	funcServersPermissions, ok := server.permissionsCache[req.Secret]
-	if !ok {
-		return rsp, aoserrors.New("secret not found")
-	}
+	// funcServersPermissions, ok := server.permissionsCache[req.Secret]
```

Comment:
why this code commented?

Reply:
this method is not used. left after testing